### PR TITLE
Update Italian Translations

### PR DESCRIPTION
**Pull request by serhack**
ping @erciccione @hrumag @faudi @lancillotto 
reference to (#1582)

### CODE REVIEW

File: translations/monero-core_it.ts [hunk 1]
```diff
@@ -1079,235 +1079,244 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsWallet.qml" line="311"/>
         <source>Show seed</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostra seed</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsWallet.qml" line="351"/>
         <source>Rescan wallet balance</source>
-        <translation type="unfinished"></translation>
+        <translation>Scannerizza di nuovo il bilancio del wallet</translation>
```
Comment (erciccione) [inline translations/monero-core_it.ts]: s/wallet/portafoglio

File: translations/monero-core_it.ts [hunk 2]
```diff
@@ -2121,7 +2130,7 @@ In caso di prova di spesa, non serve specificare l&apos;indirizzo del ricevente.
     <message>
         <location filename="../wizard/WizardCreateWalletFromDevice.qml" line="112"/>
         <source>Create a new wallet from hardware device</source>
-        <translation type="unfinished"></translation>
+        <translation>Crea un nuovo wallet da un dispositivo hardware</translation>
```
Comment (erciccione) [inline translations/monero-core_it.ts]: s/wallet/portafoglio

File: translations/monero-core_it.ts [hunk 3]
```diff
@@ -2426,17 +2435,17 @@ In caso di prova di spesa, non serve specificare l&apos;indirizzo del ricevente.
     <message>
         <location filename="../wizard/WizardOptions.qml" line="273"/>
         <source>Create a new wallet from hardware device</source>
-        <translation type="unfinished"></translation>
+        <translation>Crea un nuovo wallet da un dispositivo hardware</translation>
```
Comment (erciccione) [inline translations/monero-core_it.ts]: s/wallet/portafoglio

File: translations/monero-core_it.ts [hunk 4]
```diff
@@ -972,7 +972,7 @@
...
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;Questo è un semplice tracker di vendite:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Permette al tuo cliente di scansionare il codice QR per fare un pagamento (se il cliente ha un software che supporta lo scanning del QR) .&lt;/p&gt;&lt;p&gt;Questa pagina andrà in cerca nella blockchain e nella tx pool di transazioni basate sul codice QR. Se immetti una quantità, controllerà anche che la transazione abbia quella data quantità.&lt;/p&gt;&lt;p&gt;Dipende da te se vuoi accettare le transazioni ancora non confermate o meno. è probabile che vengano confermate in poco tempo, ma c'è sempre la possibilità che non vengano confermate; per transazioni più grandi, è possibile che debba aspettare per una o più conferme. &lt;/p&gt;</translation>
```
[the remark this answers is not in the document]
Comment (serhack) [reply translations/monero-core_it.ts]: Fantastic :D

File: translations/monero-core_it.ts [hunk 5]
```diff
@@ -1079,235 +1079,244 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsLayout.qml" line="59"/>
         <source>Custom decorations</source>
-        <translation type="unfinished"></translation>
+        <translation>Decorazioni custom</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsLayout.qml" line="67"/>
         <source>No Layout options exist yet in mobile mode.</source>
-        <translation type="unfinished"></translation>
+        <translation>Nessuna opzione per lo stile esiste ancora nella modalità mobile.</translation>
```
[the remark this answers is not in the document]
Comment (serhack) [reply translations/monero-core_it.ts]: :D

File: translations/monero-core_it.ts [hunk 6]
```diff
@@ -1079,235 +1079,244 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsNode.qml" line="232"/>
         <source>Uses a third-party server to connect to the Monero network. Less secure, but easier on your computer.</source>
-        <translation type="unfinished"></translation>
+        <translation>Utilizza un server di terze parti per connettersi alla rete Monero. Meno sicuro, ma più facile sul tuo computer.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="277"/>
         <source>To find a remote node, type &apos;Monero remote node&apos; into your favorite search engine. Please ensure the node is run by a trusted third-party.</source>
-        <translation type="unfinished"></translation>
+        <translation>Per trovare un nodo remoto, scrivi &apos;Monero remote node&apos; nel tuo motore di ricerca preferito. Assicurati che il nodo sia gestito da una terza parte affidabile.</translation>
```
[the remark this answers is not in the document]
Comment (serhack) [reply translations/monero-core_it.ts]: &apos; is the apostrophe. I think there's no needed to insert ' and '.

File: translations/monero-core_it.ts [hunk 7]
```diff
@@ -1166,7 +1166,7 @@ The old wallet cache file will be renamed and can be restored later.
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="232"/>
         <source>Uses a third-party server to connect to the Monero network. Less secure, but easier on your computer.</source>
-        <translation>Utilizza un server di terze parti per connettersi alla rete Monero. Meno sicuro, ma più facile sul tuo computer.</translation>
+        <translation>Utilizza un server di terze parti per connettersi alla rete Monero. Meno sicuro, ma meno intensivo peril tuo computer.</translation>
```
Comment (erciccione) [inline translations/monero-core_it.ts]: typo `peril`

**Comment by erciccione** [comment]
Great! :)
can you squash to just one commit please?

**Comment by serhack** [comment]
Ops! I forgot that. Now it should be ok.

**Pull request merged**